iis/event_logger.h

```diff
@@ -16,12 +16,14 @@
 
 #include <Windows.h>
 
+// A simple class to use to write messages to common Windows event log.
+// The class is Moveable but non-Copyable.
```

Comment:
To get more context, in our usage, it only have one instance?

Reply:
talked with Vlad, This is for future support.

Reply:
Just to keep track of the offline conversation with @azhao155 : currently we neither copy nor move EventLogger instances around but the previous implementation was inconsistent in that it would cause problems should we have tried to do so. This fix makes the class structure more consistent and ensures we correctly move it in case if we ever do.